app/src/components/market/sections/market_buy/market_buy_container.tsx

```diff
@@ -3,16 +3,22 @@ import React from 'react'
 import { MarketMakerData } from '../../../../util/types'
 
 import { MarketBuy } from './market_buy'
+import { ScalarMarketBuy } from './scalar_market_buy'
 
 interface Props {
   marketMakerData: MarketMakerData
   switchMarketTab: (arg0: string) => void
+  isScalar: boolean
 }
 
 const MarketBuyContainer: React.FC<Props> = (props: Props) => {
-  const { marketMakerData, switchMarketTab } = props
+  const { isScalar, marketMakerData, switchMarketTab } = props
 
-  return <MarketBuy marketMakerData={marketMakerData} switchMarketTab={switchMarketTab} />
+  if (isScalar) {
+    return <ScalarMarketBuy marketMakerData={marketMakerData} switchMarketTab={switchMarketTab} />
+  } else {
+    return <MarketBuy marketMakerData={marketMakerData} switchMarketTab={switchMarketTab} />
```

Comment:
maybe a suggestion for this is to make it one line for "if (cond) return component" so without curly braces and then to omit else after that and to just "return component" saving 2 lines or at least make else also one line ...